manim/mobject/types/opengl_vectorized_mobject.py

```diff
@@ -93,11 +93,9 @@ def __init__(
         **kwargs,
     ):
         self.data = {}
-        self.fill_color = fill_color
         self.fill_opacity = fill_opacity
-        self.stroke_color = stroke_color
         self.stroke_opacity = stroke_opacity
-        self.stroke_width = stroke_width
+        self.stroke_width = [stroke_width]
```

Comment:
Why did this need to be changed? Iirc `set_stroke` should be handling converting the float into a 2d-array to be placed into `self.data`

Reply:
I've no idea why I've changed it. I've removed it and everything works fine.